v4/source/posts.yaml

```diff
@@ -47,6 +47,47 @@
         '403':
           $ref: '#/responses/Forbidden'
 
+  /posts/ephemeral:
+    post:
+      tags:
+        - posts
+      summary: Create a ephemeral post
+      description: |
+        Create a new ephemeral post in a channel.
+        ##### Permissions
+        Must have `create_post_ephemeral` permission (currently only given to system adin)
+      parameters:
+        - in: body
+          name: ephemeral_post
+          description: Ephemeral Post object to send
+          required: true
+          schema:
+            type: object
+            required:
+              - user_id
+              - post
+            properties:
+              user_id:
+                type: string
+                description: The target user id for the ephemeral post
+              post:
+                type: object
+                required:
+                  - channel_id
+                  - message
+                description: Post object to create
```

Comment:
This might work, but I'll think you'll have to specify properties for `channel_id` and `message`

Reply:
Worked fine, but you are right, it should be added never the less!